cargo-concordium/src/main.rs

```diff
@@ -225,22 +275,32 @@ struct BuildOptions {
         short = "e",
         help = "Builds the contract schema and embeds it into the wasm module."
     )]
-    schema_embed:      bool,
+    schema_embed:        bool,
     #[structopt(
         name = "schema-out",
         long = "schema-out",
         short = "s",
         help = "Builds the contract schema and writes it to file at specified location."
     )]
-    schema_out:        Option<PathBuf>,
+    schema_out:          Option<PathBuf>,
     #[structopt(
         name = "schema-json-out",
         long = "schema-json-out",
         short = "j",
         help = "Builds the contract schema and writes it in JSON format to the specified \
                 directory."
     )]
-    schema_json_out:   Option<PathBuf>,
+    schema_json_out:     Option<PathBuf>,
+    #[structopt(
+        name = "schema-template-out",
+        long = "schema-template-out",
+        short = "p",
+        help = "Writes the template of the schema to file at specified location or prints the \
+                template of the schema to the console if the value `-` is used. The path has to \
+                exist while the file will be created. (expected input: \
```

Comment:
I think we are creating the parent directory

Reply:
Can we either change the text, or keep the behaviour consistent.

The behaviour should be consistent across all commands that have `out` flags.

Reply:
Changed text.